apps/frontend/src/modules/auth/components/PublicRoute.tsx

```diff
@@ -1,24 +1,42 @@
-import { GlobalSpinner } from '@/components';
-import { useViewerQuery } from '@/generated/graphql';
-import React from 'react';
-import { useNavigate } from 'react-router-dom';
-import { handleRedirect } from '../helper';
+import { GlobalSpinner } from "@/components";
+import { useSession } from "@/modules/users/hooks/useSession";
+import React, { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { handleRedirect } from "../helper";
+import { useAuthToken } from "../hooks/useAuthToken";
 
-export default function PublicRoute({children}: {children: React.ReactNode}) {
+export default function PublicRoute({
```

Comment:
@AfeezGL Why do we have `PublicRoute` and `PrivateRoute` in different directories?